src/EditorFeatures/Test/CodeFixes/ExtensionOrderingTests.cs

```diff
@@ -40,14 +40,14 @@ public void TestNoCyclesInFixProviders()
             var actualOrder = ExtensionOrderer.Order(csharpProviders).ToArray();
             Assert.True(actualOrder.Length > 0);
             Assert.True(actualOrder.IndexOf(p => p.Metadata.Name == PredefinedCodeFixProviderNames.AddImport) <
-                actualOrder.IndexOf(p => p.Metadata.Name == PredefinedCodeFixProviderNames.RenameTracking));
+                actualOrder.IndexOf(p => p.Metadata.Name == PredefinedCodeFixProviderNames.FullyQualify));
```

Comment:
Was the point of this test to assert rename tracking was always highest? I wouldn't have imagined that we say fully qualify is higher than add import?